web-server/src/controllers/auth/changeGeneral.ts

```diff
@@ -0,0 +1,123 @@
+import { Request, Response, NextFunction } from 'express';
+import validator from 'validator';
+
+import { AppDataSource } from 'orm/data-source';
+import { User } from 'orm/entities/users/User';
+import { CustomError } from 'utils/response/custom-error/CustomError';
+
+import { Countries } from '../../orm/entities/users/types';
+
+export const changeGeneral = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Have you considered moving this declaration to `src/controller/users/update` or `src/controller/users/edit` or whatever name it is that we're using?